src/PrestaShopBundle/Form/Admin/Improve/Shipping/Carrier/GeneralSettings.php

```diff
@@ -52,6 +53,8 @@ public function __construct(
 
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
+        $maximumFileSize = (int) str_replace('M', '', ini_get('upload_max_filesize'));
```

Comment:
Actually to match with the CQRS implementation you should use `PrestaShop\PrestaShop\Adapter\Carrier\Validate\CarrierValidator::MAX_IMAGE_SIZE_IN_BYTES`